src/js/components/Modal/Modal.stories.tsx

```diff
@@ -31,7 +31,37 @@ stories.add('Default', () => {
   )
 });
 
-stories.add('With default CTA', () => {
+stories.add('Closing triggered by async action', () => {
+  const [sending, setSending] = React.useState(false);
+
+  return (
+    <>
+      <Button
+        label='Test Modal'
+        onClick={() => {
+          setTimeout(() => {
+            setSending(false);
+          }, 5000);
+
+          setSending(true);
+        }}
+      />
+
+      <Modal
+        heading={<div className="u-textCenter">Self closing modal</div>}
```

Comment:
Please use single quotes

Reply:
Created https://github.com/teamsnap/teamsnap-ui/issues/691 to help prevent minor syntax issues like this in the future. I'd love for you to take that and run with it if you'd like to, Billy!

Reply:
Double quotes changed to single in the latest check-in